plugins/modules/keycloak_authz_custom_policy.py

```diff
@@ -0,0 +1,211 @@
+      API. Custom Javascript policies are only available if a client has Authorization enabled and if
+      they have been deployed to the Keycloak server as JAR files.
+
+    - This module requires access to the REST API via OpenID Connect; the user connecting and the realm
+      being used must have the requisite access rights. In a default Keycloak installation, admin-cli
+      and an admin user would work, as would a separate realm definition with the scope tailored
+      to your needs and a user having the expected roles.
+
+    - The names of module options are snake_cased versions of the camelCase options used by Keycloak.
+      The Authorization Services paths and payloads have not officially been documented by the Keycloak project.
+      U(https://www.puppeteers.net/blog/keycloak-authorization-services-rest-api-paths-and-payload/)
+
+attributes:
+    check_mode:
+        support: full
+    diff_mode:
+        support: none
+
+options:
+    state:
+        description:
+            - State of the custom policy.
+            - On V(present), the custom policy will be created (or updated if it exists already).
+            - On V(absent), the custom policy will be removed if it exists.
+        choices: ['present', 'absent']
+        default: 'present'
+        type: str
+    name:
+        description:
+            - Name of the custom policy to create.
+        type: str
+        required: true
+    policy_type:
+        description:
+            - The type of the policy. This must match the name of the custom policy deployed to the server.
+            - Multiple policies pointing to the same policy type can be created, but their names have to differ.
+        type: str
+        required: true
+    client_id:
+        description:
+            - The V(clientId) of the Keycloak client that should have the custom policy attached to it.
+            - This is usually a human-readable name of the Keycloak client.
+        type: str
+        required: true
+    realm:
+        description:
+            - The name of the Keycloak realm the Keycloak client is in.
+        type: str
+        required: true
+
+extends_documentation_fragment:
+    - community.general.keycloak
+    - community.general.attributes
+
+author:
+    - Samuli Seppänen (@mattock)
+'''
+
+EXAMPLES = '''
+- name: Manage Keycloak custom authorization policy
+  community.general.keycloak_authz_custom_policy:
+    name: OnlyOwner
+    state: present
+    policy_type: script-policy.js
+    client_id: myclient
+    realm: myrealm
+    auth_keycloak_url: http://localhost:8080/auth
+    auth_username: keycloak
+    auth_password: keycloak
+    auth_realm: master
+'''
+
+RETURN = '''
+msg:
+    description: Message as to what action was taken.
+    returned: always
+    type: str
+
+end_state:
+    description: Representation of the custom policy after module execution.
+    returned: on success
+    type: complex
```

Comment:
One last thing (sorry I overlooked that), please don't use `type=complex`. Either use
```suggestion
    type: dict
```
or
```suggestion
    type: list
    elements: dict
```
whatever is correct.

Reply:
It seems to be `type: dict`. I cannot push anything to your branch, so you will have to do this yourself.

If you do so today I can merge this for 7.4.0 (to be released tomorrow).

Reply:
Too late, obviously :smile:. Want me to update the version_added thing in both PRs?

Reply:
Yes please :) The next minor release will be 7.5.0.